flatpak/org.nanuc.Axolotl.yml

```
@@ -67,14 +65,14 @@ modules:
         # Cargo offline mode
         CARGO_NET_OFFLINE: "true"
     build-commands:
-      # Install the packages from our offline cache.
-      - npm --prefix=axolotl-web --offline install
+      # Install the packages from the offline cache
+      - yarn --cwd=axolotl-web --offline --verbose install
```

Does this still work
```suggestion
      - yarn --cwd=axolotl-web --verbose install
```

---

Because we configure this per .yarnrc, I do follow this thinking. Pushed a commit for this change first. Now wait 2h and see if it worked

---

I think for most changes it is good enough to just wait the 20min for x86_64 Flatpak. Seems like it worked.